src/core/directives/google-map.ts

```diff
@@ -79,6 +79,11 @@ export class SebmGoogleMap implements OnChanges, OnInit {
   zoom: number = 8;
 
   /**
+   * Enables/disables if map is draggable.
+   */
+  draggable: boolean = true;
```

Comment:
Nice. so you can have translation on inputs like this?
`'draggable: mapDraggable'`
mapDraggable would be mapped to draggable